memeolist/app/src/main/java/org/aerogear/android/app/memeolist/ui/MemeListActivity.java

```diff
@@ -181,9 +181,8 @@ public void onResult(Response<MemeAddedSubscription.Data> response) {
                             }
                             displayError(R.string.error_subscribe_meme_creation);
                         } else {
-                            MemeAddedSubscription.MemeAdded node = response.data().memeAdded();
-                            Meme newMeme = new Meme(node.id(), node.photourl(), node.owner());
-                            memes.add(0, newMeme);
+                            MemeAddedSubscription.MemeAdded meme = response.data().memeAdded();
+                            memes.add(0, Meme.from(meme));
```

Comment:
Nothing too relevant, but this nomenclature seems rather confusing (meme from meme).

Reply:
@josemigallas Both are a meme, one from the Apollo client model, the other one from the app model. Anyway, feel free to send a PR if you have better idea